### PR TITLE
docs: add oss standard files/resources

### PR DESCRIPTION
**Pull request by ianaya89**
Add Checkly open-source best practices:

- SECURITY
- LICENSE
- CoC
- Issue templates
- Helper GH actions
- Remove sonar stuff (until we enable it)
- Readme tweaks

@akosveres let me know your thoughts on this. This is the format we use to have in our OSS projects but of course, we are open to ideas, feedback, or whatever is considered worth changing.

### CODE REVIEW

**Comment by ianaya89** [comment]
@akosveres if you think is ok, I can remove that

**Review by akosveres — APPROVED** [review]
LGTM

**Comment by ianaya89** [comment]
@akosveres that is a fair point, I need to double-check. Can you point me to the SDK repo?

**Comment by akosveres** [comment]
Okay, sounds good to me.

File: README.md [hunk 1]
```diff
@@ -1,20 +1,13 @@
...
-## Development
-
-Sources used for kick starting this project:
-* https://sdk.operatorframework.io/docs/building-operators/golang/tutorial/
-* https://kubernetes.io/blog/2021/06/21/writing-a-controller-for-pod-labels/
-* https://github.com/checkly/checkly-go-sdk
-* https://docs.okd.io/latest/operators/operator_sdk/golang/osdk-golang-tutorial.html
-* https://sdk.operatorframework.io/docs/building-operators/golang/advanced-topics/#external-resources
-* https://book.kubebuilder.io/cronjob-tutorial/writing-tests.html
+A kubernetes operator for [checklyhq.com](https://checklyhq.com). This operator can create Checkly checks and groups based of kubernetes CRDs and Ingress object annotations.
```
Comment (danielpaulus) [inline README.md:10]: #nit: should be "based on"

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+* [ ] APIs
+* [ ] Config
+* [ ] External
+* [ ] Test
+* [ ] Docs
+* [ ] Tooling
+* [ ] Other
+
+## Pre-Requisites
+* [ ] Go code is formatted with `go fmt`
```
Comment (danielpaulus) [inline .github/pull_request_template.md:12]: we should just make this a part of some github action and go fmt as a pr check.

**Review by danielpaulus — CHANGES_REQUESTED** [review]
Good stuff! just a nitpick and request for adding a small ci job rather than relying on contributors to run go fmt.

File: .github/pull_request_template.md (hunk 2 above)
Comment (danielpaulus) [reply .github/pull_request_template.md:12]: never mind, go test does that automatically. I wonder if this is then really needed it the template. But have no strong opinion about it.

**Pull request closed without merging**